spec/controllers/proposals_controller_spec.rb

```diff
@@ -173,6 +173,24 @@
           expect(query[2].id).to be(single.id)
         end
       end
+
+      it "returns JSON for preview count" do
+        login_as(user)
+        proposals = 3.times.map do |i|
+          wo = create(:test_client_request, project_title: "Work Order #{i}")
+          wo.proposal.update(requester: user)
+          wo.proposal.reindex
```

Comment:
refresh is not the same as reindex.

reindex pushes content to the ES server.

refresh tells the ES to commit changes to disk and re-open its searchers to make the reindex changes visible. It's like calling `COMMIT` on a db transaction.